cmd/fyne/package-darwin.go

```diff
@@ -0,0 +1,61 @@
+package main
```

Comment:
Is this file supposed to only build on darwin? If so the format would be package_darwin.go

Reply:
ignore this, I realised now it's just grouping :)

Reply:
It is unusual to have a hyphen in the name, typically an underscore is used

Reply:
Using an underscore would trigger conditional compilation but this is simply organising code per platform, excluding the methods would break the current implementation.